app/boards/shields/sofle_rgb_keyhive/sofle_rgb_keyhive.conf

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+# Uncomment the following line to enable the Sofle OLED Display
+# CONFIG_ZMK_DISPLAY=y
+
+# Uncomment the following two lines to disable the underglow LEDs
+# CONFIG_ZMK_RGB_UNDERGLOW=n
```

Comment:
Actually one more thing, this seems wrong, doesn't it? I don't see anywhere else that enables these and I think it'd be a good idea to stick to convention (i.e. add commented lines to *enable* the settings).